addons/web/views/webclient_templates.xml

```diff
@@ -100,7 +100,7 @@
             <link rel="stylesheet" href="/web/static/src/css/data_export.css"/>
             <link rel="stylesheet" href="/base/static/src/css/modules.css"/>
 
-            <link rel="stylesheet" href='/website/static/src/less/import_bootstrap.less'/>
+            <link rel="stylesheet" href='/web/static/src/less/import_bootstrap.less'/>
```

Comment:
You need to create separate git branch and add there only one commit, that fix this issue only.